admin/app/com/lucidchart/piezo/admin/controllers/TriggerFormHelper.scala

```diff
@@ -127,20 +132,56 @@ class TriggerFormHelper(scheduler: Scheduler) extends JobDataHelper {
       "jobGroup" -> nonEmptyText(),
       "jobName" -> nonEmptyText(),
       "description" -> text(),
-      "simple" -> optional(mapping(
-        "repeatCount" -> number(),
-        "repeatInterval" -> number()
-      )(simpleScheduleFormApply)(simpleScheduleFormUnapply)),
-      "cron" -> optional(mapping(
-        "cronExpression" -> nonEmptyText().verifying(validCronExpression)
-      )(cronScheduleFormApply)(cronScheduleFormUnapply)),
+      "simple" -> optional(
+        mapping(
+          "repeatCount" -> number(),
+          "repeatInterval" -> number(),
+        )(simpleScheduleFormApply)(simpleScheduleFormUnapply),
+      ),
+      "cron" -> optional(
+        mapping(
+          "cronExpression" -> nonEmptyText().verifying(validCronExpression),
+        )(cronScheduleFormApply)(cronScheduleFormUnapply),
+      ),
       "job-data-map" -> jobDataMap,
       "triggerMonitoringPriority" -> nonEmptyText(),
-      "triggerMaxErrorTime" -> number()
-    )(triggerFormApply)(triggerFormUnapply).verifying("Job does not exist", fields => {
-      scheduler.checkExists(fields._1.getJobKey)
-    }).verifying("Max time between successes must be greater than 0", fields => {
-      fields._3 > 0
-    })
+      "triggerMaxErrorTime" -> of(MaxSecondsBetweenSuccessesFormatter).verifying(Constraints.min(0)),
+    )(triggerFormApply)(triggerFormUnapply)
+      .verifying(
+        "Job does not exist",
+        fields => {
+          scheduler.checkExists(fields._1.getJobKey)
+        },
+      ),
   )
 }
+
+object MaxSecondsBetweenSuccessesFormatter extends Formatter[Int] {
+  override val format = Some(("format.triggerMaxErrorTime", Nil))
+  override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], Int] = {
+    for {
+      maxSecondsBetweenSuccesses <- parsing(_.toInt, "Numeric value expected", Nil)(key, data)
+      maxIntervalTime <- {
+        if (data.contains("cron.cronExpression")) {
+          parsing(expr => CronHelper.getMaxInterval(expr), "try again.", Nil)(
```

Comment:
should we use better error messages for these?

Reply:
"try again" is appended to the error message that is generated from the `parsing` method. 
This is what it looks like in practice:
![image](https://github.com/lucidsoftware/piezo/assets/8989774/fd31c8ba-0eac-4856-81b8-31b008de48b4)